Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
+gem 'rexml' if RUBY_VERSION > "3"
```

Comment:
We should add this to `.gemspec`.

Reply:
@kou Thanks, that's better, and I changed it, now.